ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -23,7 +22,11 @@ do
     result=$(echo "$job_check" | jq -r '.result')
     if [ $result != "null" ]; then
 	sleep $((RANDOM % 61 + 60))
-        break
+	job_check=$(curl -k -s --resolve "$endpoint_resolve" "$job_url/api/json")
```

Comment:
This is a copy paste of the code above. Could you think of a way to do it without copying?

Reply:
Also this is an infinite loop if the exit condition is never met.

Reply:
okay